src/js/a11y/020-tabbing.js

```diff
@@ -21,29 +21,22 @@ AddEvent( document.querySelector( '.js-tabbing-switch' ), 'click', function( eve
 });
 
 
-/**
- * Event listener for when the user scrolls or uses the tab key to stop Tab() function
- */
-AddEvent( document.querySelector( '.tabbing-frame' ), 'keydown', function( event, $this ) {
-	StopTab( interval, document.querySelector( '.js-tabbing-switch' ) )
-});
-
-
 /**
  * Move the tab through each element with the .js-focus-me class
  */
-function Tab() {
+ function Tab() {
 	var items = document.querySelectorAll( '.js-focus-me' );
-	var loop = 0;
-
+	var loop = 0; 
```

Comment:
whitespace issues. Perhaps you should use prettier again?
```suggestion
	var loop = 0;
```